packages/ember-metal/lib/map.js

```diff
@@ -197,24 +183,23 @@ OrderedSet.prototype = {
   */
   toArray() {
     return this.list.slice();
-  },
+  }
 
   /**
     @method copy
     @return {Ember.OrderedSet}
     @private
   */
   copy() {
-    let Constructor = this.constructor;
-    let set = new Constructor();
+    let set = new OrderedSet();
```

Comment:
This change breaks inheritance, any child classes will now start returning `OrderSet` instead of `ThingThatExtendsOrderedSet`.

Reply:
thanks good catch 👍 will fix it